synapse/lib/cmdr.py

```diff
@@ -5,6 +5,8 @@
 cmdsbycell = {
     'cortex': (
         s_cmds_cortex.StormCmd,
+        s_cmds_cortex.PsCmd,
```

Comment:
shouldn't these on on the base 'cell' type? does getCellType() event support a class hierarchy to support that type of inheritance?

Reply:
No, not right now.  And i'd like to tweak the way a bunch of this works in another delta, so i just did it this way to get it rolling :)